audit/projects/kubernetes-public/services/pubsub/subscriptions.json

```diff
@@ -1,13 +0,0 @@
-[
-  {
-    "ackDeadlineSeconds": 10,
-    "expirationPolicy": {
-      "ttl": "2678400s"
-    },
-    "filter": "attributes.eventType = \"OBJECT_FINALIZE\"",
-    "messageRetentionDuration": "604800s",
-    "name": "projects/kubernetes-public/subscriptions/k8s-infra-kettle-staging",
```

Comment:
I thought it was a glitch of the audit script but the subscription is deleted and I can't find a trace of this action. @spiffxp Any idea ?

Reply:
I still don't know what happened. I'll investigate this more deeply in 2022.